### ISSUE TITLE
Favorite Stores

### ISSUE THREAD

**Issue by ChrisASearles** [ChrisASearles/CompLinks#47]

**Comment by cgladue** [comment]
@robgpeak is this going to be in the v1.0 release ?  or something down the road ?

**Comment by robgpeak** [comment]
Not sure I know what you mean.  It's in place now in the main nav.  Meaning you can see featured stores in the main navigation, and you can also see featured stores in the home page.  What else is it supposed to do?

**Comment by cgladue** [comment]
@robgpeak favorite stores, like a user being able to click a heart icon or something to set it as a favorite store, not featured store ;)  you need another cup of coffee i think

**Comment by robgpeak** [comment]
Oh shit.  Sorry about that.

You tell me?  What would it take for you to program that?

As for coffee, I never drink the stuff... Maybe I should start.

**Comment by cgladue** [comment]
it wouldnt take me long at all, but Chris and derek would need to add the heart icons into their design somehow and add a place to see your favorite stores somewhere in the profile page i suppose

**Comment by robgpeak** [comment]
Well as long as all your other homework is done, you can build the portions that you need to, and then Derek and Chris can put the front end stuff in place when they get to it.

I know we've spoke about different version releases of the software, but realistically, it will be an ongoing process virtually forever.  Meaning, we will always be making improvements and upgrades to the product, and this is just another one...

**Comment by cgladue** [comment]
yea of course, i just mean v1.0 as in the initial offering, i would expect there will be changes often which is a good thing

**Comment by robgpeak** [comment]
Well what does v1.0 actually mean.  Do you draw a hard line in the sand and say, these are the only features we will include in v1.0, and then if you add favorite stores, is the release then v1.1?  

I am asking this question in all seriousness

**Comment by cgladue** [comment]
there wouldnt be any version numbers it was just a theoretical 1.0 just meaning the first version as in the day you go live actually accepting customers on the site.  in normal development there would be a code freeze, peer review, etc before a "release" but i dont like that much structure personally i like to release on an ongoing basis no set guidelines other then testing needs to be done beforehand.

**Comment by robgpeak** [comment]
I agree with that philosophy, as long as if you make a change, release it to the public, and find out something is NOT working as expected, you can roll back to a previous version where everything was working.

**Comment by cgladue** [comment]
yea with version control we can go back to any prior file version.

**Comment by robgpeak** [comment]
Well then, I think as long as you keep very detailed records of every version including what feature specifically was changed, as small as it may be, you should be OK in terms of being able to roll back to a fully working version if need be.

Having said that, I CAN see the reasoning behind a code freeze and internal review before releasing it live.  I'm sure we can find the right balance.
